extendedset/pom.xml

```diff
@@ -58,7 +58,7 @@
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
-      <version>4.8.1</version>
+      <version>4.12</version>
```

Comment:
Thanks for cleaning this up! I looked through all uses of junit in the poms, and it looks like this is the only sub-module that explicitly specifies the version. Can we exclude the version here so that it will pick up the version specified in the top level pom?

Similar comment for JUnitParams below

Reply:
Yes. It's better.  I have made suggested changes